src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -275,7 +275,10 @@ public WorkflowRun(WorkflowJob job, File dir) throws IOException {
             if (!auth.equals(ACL.SYSTEM)) {
                 String name = auth.getName();
                 if (!auth.equals(Jenkins.ANONYMOUS)) {
-                    name = ModelHyperlinkNote.encodeTo(User.get(name));
+                    User user = User.getById(name, false);
```

Comment:
This appears to be a separate fix... but a worthwhile one that avoids an NPE potentially.